salt/modules/dockerio.py

```diff
@@ -648,8 +675,9 @@ def create_container(image,
         }
         __salt__['mine.send']('docker.get_containers', host=True)
         return callback(status, id_=container, comment=comment, out=out)
-    except Exception:
+    except Exception, e:
```

Comment:
If it turns out to be too difficult to account for all the errors coming to this point, then don't worry about it too much.